jck/build.xml

```diff
@@ -119,43 +190,53 @@
 		</if>
 		<copy todir="${DEST}">
 			<fileset dir="${basedir}">
-				<exclude name="jck_root/"/>
-				<exclude name="README.md"/>
+				<exclude name="jck_root/" />
+				<exclude name="README.md" />
 			</fileset>
 		</copy>
-		<copy todir="${JCK_ROOT}/">
+		<copy todir="${env.JCK_ROOT}/">
 			<fileset dir="${basedir}/../systemtest_prereqs/" includes="**" />
 		</copy>
 	</target>
 
 	<target name="build">
-		<echo>JCK_ROOT is set to: ${JCK_ROOT}</echo>
 		<if>
-			<isset property="JCK_ROOT" />
+			<isset property="env.JCK_ROOT" />
 			<then>
+				<echo>=== env.JCK_ROOT is set to ${env.JCK_ROOT} ===</echo>
 				<if>
-					<not>
-						<equals arg1="${JCK_ROOT}" arg2="" />
-					</not>
+					<isset property="env.JCK_GIT_REPO" />
 					<then>
-						<echo>=== JCK_ROOT is set to ${JCK_ROOT} ===</echo>
+						<echo>=== env.JCK_GIT_REPO is set to ${env.JCK_GIT_REPO} ===</echo>
+						<echo>start staging jck materials</echo>
+						<antcall target="stage_jck_material" inheritall="true" />
 						<echo>start building jck project</echo>
 						<antcall target="dist" inheritall="true" />
+						<echo>start compiling jck natives</echo>
+						<antcall target="compile_jck" inheritall="true" />
 					</then>
+					<else>
+						<echo>start building jck project</echo>

```

Comment:
nitpicking: I find echo msg `start building jck project` and `start compiling jck natives` are confusing. Could we change `start building jck project` to be more descriptive? i.e., prepare/configure stf test framework?

Reply:
updated